tools/enumerate-devices/readme.md

```diff
@@ -21,7 +21,16 @@ Product Id                    :     0AD3
 
 |Flag   |Description   |
 |---|---|
-|`-s`|Provide short summary, one line per device|
-|`-o`|List supported device controls and options|
-|`-c`|Provide calibration (Intrinsic/Extrinsic) information|
-| None| List supported streaming modes|
+|`-s`|Provide a one-line summary, one line per device|
+|`-S`|Extends `-s` by providing a short summary for each device|
+|`-o`|List supported device controls, options and streaming modes|
+|`-c`|Provide calibration (Intrinsic/Extrinsic) and steraming modes information|
+|`-p`|Enumerate streams contained in ROSBag record file. Usage `-p <rosbag_full_path>`|
+| None| The default mode. Equivalent to `-S` plus the list of all the supported streaming profiles|
+
+The options `-o`, `-c` `-p` are additive and can concatenated:
```

Comment:
can **be** concatenated
Though I'd replace it with 'combined'.
Also, missing comma between -c and -p

Reply:
Done